app/scripts/modules/core/src/managed/versionsHistory/VersionContent.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import ContentLoader from 'react-content-loader';
+// import ContentLoader from 'react-content-loader';
```

Comment:
as a general note, we usually discuss with the group before adding new deps
this one seems teeny though, so I'm 👍

Reply:
Good to know, thanks